src/modules/leaderboard.ts

```diff
@@ -31,6 +31,51 @@ async function updateLeaderboardRow(address: string, property: string, value: nu
   return true;
 }
 
+async function updateLeaderboardNfts() {
+  const name = `updateLeaderboardNfts`;
+  logger.info(name);
+
+  try {
+    const proposals = await db.merge_proposals.findAll({
+      include: [
+        {
+          association: "issue",
+          required: true,
+          where: {
+            state: "closed",
+          },
+        },
+      ],
+    }).then(v => v.filter(({ contractId, issue: { merged }}) => contractId?.toString() === merged))
```

Comment:
No need of filtering, you can add this condition to the query, as we are doing here https://github.com/taikai/webnetwork-events/blob/64a0fd780df0eac7cf18428186d3b16113bb4c5f/src/modules/leaderboard.ts#L157